instagram-app/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './comment.css';
+
+
+const Comment = props => {
+    return (
+      <div>
+        <span><strong>{props.comment.username}</strong></span> {props.comment.text}
+      </div>
+    );
+  };
+
+Comment.propTypes ={
+    comment: PropTypes.shape({
+        text: PropTypes.string.isRequired,
+        username: PropTypes.string.isRequired
```

Comment:
When a comment is typed, there appears a warning that indicates *comment.username* is marked as required but its value is null.